config.default.json

```diff
@@ -111,6 +111,7 @@
     "difficulty": 22,
     "numThreads": 1,
     "timeout": "1m"
+
```

Comment:
Should this be added here? `"parentsRefreshInterval": "300ms"`
Or do we just rely on the default value?

Reply:
Yeah, we thought that by specifying that in the default value was easier so we don't care to set that here